pkg/utils/x509.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"github.com/openshift/library-go/pkg/crypto"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func ComplianceOperatorRootCA(certname string, expire int) ([]byte, []byte, error) {
+	config, err := crypto.MakeSelfSignedCAConfig(certname, expire)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return config.GetPEMBytes()
+}
+
+func NewServerCert(caCert, caKey []byte, certname string, expire int) ([]byte, []byte, error) {
```

Comment:
Do you plan on using the expire timeout? If a constant is to be used, do you think it makes sense to define it in the code?

Reply:
Done